quickstart-linux.md

```diff
@@ -6,16 +6,26 @@ Installing the emulator
 
 ### Getting the emulator
 
-Obtain the X16 emulator by downloading an official release from the 'releases' tab in Github
-(note: currently no recent official releases are available).
-Alternatively, download a development build which are automatically created by github every time
-an update gets made to the emulator. You find these here: https://github.com/X16Community/x16-emulator/actions/workflows/build.yml?query=branch%3Amaster
-You'll need to be signed in into Github to access these files. Look for the Artifact called 'X16-Emu Linux 64-bit'.
+Obtain the X16 emulator by downloading it from the 'releases' tab in Github:
+https://github.com/X16Community/x16-emulator/actions/workflows/build.yml?query=branch%3Amaster
```

Comment:
I would recommend pointing to the releases page now that we have one.  People won't need to be logged into GitHub for these.  IMO the quick start shouldn't need to mention the CI builds.

Reply:
https://github.com/X16Community/x16-emulator/releases

Reply:
I didn't notice the releases page before. I shall update the PR